### PR TITLE
Checkpoint SandBlast appliance

### PR DESCRIPTION
**Pull request by Itay4**
Towards session, work in progress

### CODE REVIEW

File: Integrations/integration-SandBlastAppliance.yml [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+  type: 8
+  required: false
+script:
+  script: |-
+    var server = params.server;
+    var insecure = params.insecure;
+    var proxy = params.proxy;
+    var version = params.version;
+    var requestTemplate = JSON.stringify({request:[{features:['te']}]});
+    var base = server.replace(/[\/]+$/, '') +':18194/tecloud/gw/' + version + '/file/';
```
Comment (anara123) [inline Integrations/integration-SandBlastAppliance.yml]: Isn't it better the port to be configurable together with the server URL?

File: Integrations/integration-SandBlastAppliance.yml [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+        'sb-query': 'query',
+        'sb-upload': 'upload',
+        'sb-download': 'download',
+        'sandblast-query': 'query',
+        'sandblast-upload': 'upload',
+        'sandblast-download': 'download'
+    };
+
+    var sendRequest = function(url, body, queryName) {
+        log('URL IS: '+base + url)
```
Comment (anara123) [inline Integrations/integration-SandBlastAppliance.yml]: remove those logs. You can replace them by logDebug

File: Integrations/integration-SandBlastAppliance.yml [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+            args.features = [args.features];
+        }
+        var request = {request: args};
+        return JSON.stringify(request);
+    };
+
+    var raw;
+    switch (command) {
+        case 'test-module':
+            if (sendRequest(urlDict['sb-query'], createRequest({md5:'31335bc6357f0eb6e3370803b9a6a318'}), 'test')) {
```
Comment (anara123) [inline Integrations/integration-SandBlastAppliance.yml]: What is 31335bc6357f0eb6e3370803b9a6a318 hash?

File: Integrations/integration-SandBlastAppliance.yml [hunk 4]
```diff
@@ -0,0 +1,450 @@
...
+                        }
+                        delete args[keys[i]];
+                    }
+                }
+                if (Object.keys(feature).length) {
+                    args['te'] = feature;
+                }
+        }
+        if (args.features === 'all') {
+            args.features = ['te', 'av', 'extraction'];
```
Comment (anara123) [inline Integrations/integration-SandBlastAppliance.yml]: Don't we need only `te` feature only here?

File: Integrations/integration-SandBlastAppliance.yml (hunk 1 above)
Comment (Itay4) [reply Integrations/integration-SandBlastAppliance.yml]: According to the API, this is the only port to work on prem.

File: Integrations/integration-SandBlastAppliance.yml (hunk 4 above)
Comment (Itay4) [reply Integrations/integration-SandBlastAppliance.yml]: No, we need them all.

File: Integrations/integration-SandBlastAppliance.yml (hunk 3 above)
Comment (Itay4) [reply Integrations/integration-SandBlastAppliance.yml]: Just some hash for the test

**Pull request merged**